examples/complete/schema.graphql

```diff
@@ -12,10 +12,16 @@ type User {
   name: String
 }
 
+type Entity {
```

Comment:
Let's remove `Entity` from schema and use existing types to illustrate the functionality of JS functions on type `User`, for example.